draft-ietf-quic-transport.md

```diff
@@ -2759,10 +2759,16 @@ ordered byte-stream.  Data received out of order MUST be buffered for later
 delivery, as long as it is not in violation of the receiver's flow control
 limits.
 
-An endpoint MUST NOT send data on any stream without ensuring that it is within
-the data limits set by its peer.  The cryptographic handshake stream, Stream 0,
-is exempt from the connection-level data limits established by MAX_DATA.  Stream
-0 is still subject to stream-level data limits and MAX_STREAM_DATA.
+An endpoint MUST NOT send data on any stream without ensuring that it
+is within the data limits set by its peer.  The cryptographic
+handshake stream, Stream 0, is exempt from the connection-level data
+limits established by MAX_DATA. Data on stream 0 other than the
+TLS ClientHello is still subject to stream-level data limits and
```

Comment:
editorial nit: Can you say "initial cryptographic message from the client" instead of TLS ClientHello, for consistency (here and below)?